src/Test.UnitTests.Sarif/TestData/SarifCurrentToVersionOneVisitor/Inputs/OneRunWithNotificationTime.sarif

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-09-26",
+  "version": "2.0.0-csd.2.beta.2018-09-26",
+  "runs": [
+    {
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "invocations": [
+        {
+          "commandLine": "CodeScanner @collections.rsp",
+          "toolNotifications": [
+            {
+              "id": "CTN9999",
+              "ruleId": "C2152",
+              "physicalLocation": {
+                "fileLocation": {
+                  "uri": "file:///home/buildAgent/src/crypto/hash.cpp"
+                }
+              },
+              "message": {
+                "text": "Exception evaluating rule \"C2152\". Rule disabled; run continues."
+              },
+              "level": "error",
+              "threadId": 52,
+              "time": "2016-07-16T14:18:43.119Z",
```

Comment:
>"time" [](start = 14, length = 6)

this will internally transform to utctime during prerelease transformation (not visible in output because the test case is really for v2 -> v1 conversion. so, internally, first this file is upgraded to rtm schema version, then converted to v1. The fact that we see the time field in output (v1) file, is proof that transformation was applied correctly.